### PR TITLE
Add AWS sample to bootstrap the remote backend for OpenTofu to documentation

### PR DESCRIPTION
**Pull request by LumberjackUsingMath**
Adding a reference to the AWS sample which provides an example to bootstrap the Amazon S3 backend for OpenTofu following best practices. 

closes  https://github.com/opentofu/opentofu/issues/1339

Resolves #

## Target Release

1.7.0

### CODE REVIEW

File: website/docs/language/settings/backends/s3.mdx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-sidebar_label: s3
+sidebar_label: Amazon S3
```
Comment (Yantrio) [inline website/docs/language/settings/backends/s3.mdx:2]: I would prefer to keep this as just "s3" if possible as we may in the future try and support other s3-compatible backends officially.

**Review by Yantrio — CHANGES_REQUESTED** [review]

File: website/docs/language/settings/backends/s3.mdx [hunk 2]
```diff
@@ -18,6 +18,12 @@ It is highly recommended that you enable
 on the S3 bucket to allow for state recovery in the case of accidental deletions and human error.
 :::
 
+## Bootstrap the backend using Amazon CloudFormation
+
+Amazon provides a sample repository for bootstrapping the backend with Amazon CloudFormation, accessible at the following link: [bootstrap-amazon-s3-remote-backend-for-open-tofu
+](https://github.com/aws-samples/bootstrap-amazon-s3-remote-backend-for-open-tofu).
```
Comment (virtualroot) [inline website/docs/language/settings/backends/s3.mdx:24]: This repo is gone (404)

**Comment by janosdebugs** [comment]
Hi @LumberjackUsingMath thank you for the PR and the discussion. Personally, I'm not a fan of including vendor-specific links in the official documentation, as you can see we didn't include the Spacelift link in the documentation either, even though it is one of the companies sponsoring the OpenTofu development.

If this were to be included, I would either like to see a comprehensive list of resources for cloud providers such as Amazon, DigitalOcean, IONOS, etc, as well as on-prem solutions such as Minio or Synology C2. OpenTofu is a vendor-neutral tool and the documentation should reflect that.As far as the naming is concerned, S3 (or S3-compatible object storage) is a pretty well-known term in the industry, I don't think it needs more explanation.

File: website/docs/language/settings/backends/s3.mdx (hunk 2 above)
Comment (janosdebugs) [reply website/docs/language/settings/backends/s3.mdx:24]: Did you copy the `)`? It works for me.

Comment (virtualroot) [reply website/docs/language/settings/backends/s3.mdx:24]: Weird, it works now. 
I did search for tofu in the `aws-samples` org with no results, but I see it now.

**Pull request closed without merging**